tests/test_input_module.py

```diff
@@ -1,6 +1,7 @@
 """
 Test module for the inputs module of the quicfire_tools package.
 """
+from lib2to3.refactor import get_all_fix_names
```

Comment:
Can we remove this import?

Reply:
Yes sorry, sometimes code randomly imports things if I mistype a function. I didn't even type that line